project/api/write.py

```diff
@@ -462,7 +386,7 @@ def clearProfileImage(request):
             account.save()
 
             return HttpResponse('Image Deleted')
-        except Exception as e:
+        except Exception:
```

Comment:
Removing the variable `e` from this line may cause line **478** to fail. Rather, rename `e` to be `exception`, and update line **478** to use the `exception` variable.

Reply:
I don;t understand how the except at line 465 can affect anything after the line 466 since its returning.  Line 478 is a new try block?  Unless you mean to update line 466 to use the exception object instead of using return HttpResponseServerError(reason=str(default)).  So I could change it to be return HttpResponseServerError(reason=str(e)).

Reply:
Ah, OK. Nevermind. The code truncation makes it hard to follow the flow of the files sometimes.